corehq/motech/repeaters/_design/views/repeaters_records/reduce.js

```diff
@@ -0,0 +1 @@
+_count
```

Comment:
why remove this? I don't think there's any harm in having it and without it you can't do any aggregations.